docs/en/modules/tmr.md

```diff
@@ -34,7 +34,12 @@ To free up the resources with this timer when done using it, call [`tmr.unregist
 
 #### Example
 ```lua
-if not tmr.alarm(0, 5000, tmr.ALARM_SINGLE, function() print("hey there") end) then print("whoopsie") end
+if not tmr.create():alarm(5000, tmr.ALARM_SINGLE, function()
```

Comment:
Yes, it does:

```lua
print("before: " .. collectgarbage("count")*1024)
if not tmr.create():alarm(5000, tmr.ALARM_SINGLE, function()
  print("hey there")
end)
then
  print("whoopsie")
end
print("while running: " .. collectgarbage("count")*1024)
```

Prints 
> before: 3072
> while running: 4096
> hey there

If you run `print("after: " .. collectgarbage("count")*1024)` after the timer finished it yields

> after: 3072